partcad/src/partcad/__init__.py

```diff
@@ -85,17 +96,78 @@ def before_send(event, hint):
         return event
 
     sentry_sdk.init(
+        # FIXME: sentry_sdk.utils.BadDsn: Missing public key
         dsn=user_config.get_string("sentry.dsn"),
         release=__version__,
+        environment=user_config.get_string("sentry.environment"),
         debug=user_config.get_bool("sentry.debug"),
         shutdown_timeout=user_config.get_int("sentry.shutdown_timeout"),
-        enable_tracing=True,
-        attach_stacktrace=False,
+        enable_tracing=user_config.get_bool("sentry.enable_tracing"),
+        attach_stacktrace=user_config.get_bool("sentry.attach_stacktrace"),
         traces_sample_rate=user_config.get_float("sentry.traces_sample_rate"),
+        default_integrations=False,
+        profiles_sample_rate=1.0,
         integrations=[
             LoggingIntegration(
-                level=logging.ERROR,
+                level=logging.logging.getLevelName(
+                    user_config.get_string("sentry.integrations.LoggingIntegration.level").upper()
+                ),
```

Comment:
This is to allow to write `level: ERROR` in YAML.

P. S. IDK where is the right palace to document Sentry. Probably the "Handbook".